generate/tsconfig.module.json

```diff
@@ -1,7 +1,8 @@
 {
     "compilerOptions": {
       "outDir": "../source/js/esm/",
-      "moduleResolution": "node",
+      "moduleResolution": "node16",
+      "module": "node16",
```

Comment:
@talyguryn I haven't seen this before, where did you get this way?

Reply:
check related links in my comment https://github.com/cosinekitty/astronomy/issues/294#issuecomment-1934804842